cartography/intel/crxcavator/crxcavator.py

```diff
@@ -49,55 +52,139 @@ def call_crxcavator_api(api_and_parameters, crxcavator_api_key, crxcavator_base_
     return data.json()
 
 
-def transform_extensions(extension_json):
+def get_extensions(crxcavator_api_key, crxcavator_base_url, extensions_list):
+    """
+    Retrieves the detailed information for all the extension_id and version pairs
+    :param crxcavator_api_key: The API key to access the CRXcavator service
+    :param crxcavator_base_url: The URL for the CRXcavator API
+    :param extensions_list: list of dictonary items containing the extension_id and version pairs
+    :return: list containing all metadata for extensions
+    """
+    extensions_details = []
+    for extension in extensions_list:
+        extension_id = extension.get('extension_id')
+        version = extension.get('version')
+        name = extension.get('name')
+        try:
+            details = get_extension_details(crxcavator_api_key, crxcavator_base_url, extension_id, version)
+            if not details:
+                # we only have the name and version from group API, create minimal version
+                logger.info(f"No results returned from report API for extension {extension_id} {version}")
```

Comment:
This log message is very high volume, can we change it to debug-level, or is it useful to have at info?